project-base/storefront/components/Pages/StoreDetail/StoreDetailContent.tsx

```diff
@@ -40,10 +36,16 @@ export const StoreDetailContent: FC<StoreDetailContentProps> = ({ store }) => {
                         </div>
                     </div>
 
+                    {!!store.specialMessage && (
+                        <InfoItem>
+                            <Infobox message={store.specialMessage} />
+                        </InfoItem>
+                    )}
+
                     {!!store.description && (
                         <InfoItem>
                             <StoreHeading text={t('Store description')} />
-                            <div dangerouslySetInnerHTML={{ __html: store.description }} />
+                            <p dangerouslySetInnerHTML={{ __html: store.description }} />
```

Comment:
I would use something like this `const cleanHtml = DOMPurify.sanitize(text);` just to be safe.

Reply:
This is used across our codebase. What about creating an issue to resolve it everywhere at once?

Reply:
well, ok :)